### DISCUSSION TITLE
Ledger wallet | Migration to the cloud registry

### DISCUSSION THREAD

**Discussion by imqdee** [WalletConnect/walletconnect-monorepo#953] [Developer Support]
Hi!
I'm qdqd, Solidity engineer at Ledger 👋 

Ledger registered his wallet before the cloud registry have been released, and today we would like to update some information and add universal links to the app. Is there a process to migrate the app to the cloud registry to let us update it? Should I create a new app using the cloud registry and then publish it to delete the one already in production?

Sorry if the migration process has been explained in the past, I probably missed this communication.

In any case, thanks for your help!

**Comment by chadyj** [comment]
Hey! You can post in our Discord, and I'll follow up with you on how to migrate your registry listing to your new account.
